src/Sentry.DiagnosticSource/Internal/DiagnosticSource/EFConnectionDiagnosticSourceHelper.cs

```diff
@@ -0,0 +1,47 @@
+using Sentry.Internal.Extensions;
```

Comment:
import not needed

Reply:
That one is needed for the `GetGuidProperty` extension that's used to read the ConnectionId.